test/meadow/ingest/bucket_test.exs

```diff
@@ -1,45 +1,35 @@
 defmodule Meadow.Ingest.BucketTest do
-  use ExUnit.Case
+  use Meadow.S3Case
   alias Meadow.Ingest.Bucket
 
-  import Mox
-
-  setup :verify_on_exit!
-
   @project_folder_name "name-of-folder"
-  @bucket 'bucket-name'
+  @bucket "test-ingest"
 
   describe "Ingest Bucket" do
-    test ".create_project_folder writes an \"empty\" folder to the ingest bucket" do
-      Meadow.ExAwsHttpMock
-      |> expect(:request, 2, fn _method, _url, _body, _headers, _opts ->
-        {:ok, %{status_code: 200}}
+    setup do
+      on_exit(fn ->
+        empty_bucket(@bucket)
+        delete_bucket("nonexistent-bucket")
```

Comment:
The above lines should actually be 
```
        delete_object(@bucket, "#{@project_folder_name}/")
        delete_bucket("nonexistent-bucket")
```
Emptying the whole `test-ingest` bucket might mess up other tests.